### PR TITLE
Add plugin: cluster-group

### PR DESCRIPTION
**Pull request by u2takey**
This [plugin](https://github.com/u2takey/kubectl-clusters) allows users exec kubectl commands in multiple clusters with group.

### CODE REVIEW

**Comment by ahmetb** [comment]
Thanks. The name “clusters” is too generic for what it does. 

I initially assumed it lists clusters, turned out to be a tool running tools in multiple commands and requires a special configuration file. Can you consider alternative names?

File: plugins/clusters.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  name: clusters
+spec:
+  version: "v0.1.1"
+  platforms:
+  - uri: https://github.com/u2takey/kubectl-clusters/archive/v0.1.1.tar.gz
+    sha256: 369dcf9c83f38b0068a5176b802285bfe4cfda60b7e0b4c65841e5c354fd8bc7
+    bin: kubectl-clusters
+    files:
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
```
Comment (corneliusweig) [inline plugins/clusters.yaml]: Please include your `LICENSE` file and install that along with your binary file.

File: plugins/clusters.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: clusters
```
Comment (corneliusweig) [inline plugins/clusters.yaml]: Maybe you can take inspiration from SIMD (single instruction multiple data). For example `simc` -- _single instruction multiple contexts_.

File: plugins/clusters.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the config file clusters.ini in current dir or refer with env CLUSTERS_CONFIG.
+  description: |
+    This plugin allows users exec kubectl commands in multiple clusters with group.
+  shortDescription: Exec commands in multiple clusters with group
```
Comment (corneliusweig) [inline plugins/clusters.yaml]:
```suggestion
  shortDescription: Exec commands across a group of contexts
```

I think `context` is the proper term here.

File: plugins/clusters.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    files:
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the config file clusters.ini in current dir or refer with env CLUSTERS_CONFIG.
+  description: |
+    This plugin allows users exec kubectl commands in multiple clusters with group.
```
Comment (corneliusweig) [inline plugins/clusters.yaml]:
```suggestion
    This plugin performs the same command across a group of multiple contexts.
```

File: plugins/clusters.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    sha256: 369dcf9c83f38b0068a5176b802285bfe4cfda60b7e0b4c65841e5c354fd8bc7
+    bin: kubectl-clusters
+    files:
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the config file clusters.ini in current dir or refer with env CLUSTERS_CONFIG.
```
Comment (corneliusweig) [inline plugins/clusters.yaml]: You should also mention that you require a python runtime. You should also mention the minimum version required and potentially non-standard libraries that need to be installed.

**Comment by u2takey** [comment]
> Thanks. The name “clusters” is too generic for what it does.
> 
> I initially assumed it lists clusters, turned out to be a tool running tools in multiple commands and requires a special configuration file. Can you consider alternative names?

renamed clusters to `cluster-group`

File: plugins/clusters.yaml (hunk 1 above)
Comment (u2takey) [reply plugins/clusters.yaml]: LICENSE added

File: plugins/clusters.yaml (hunk 5 above)
Comment (u2takey) [reply plugins/clusters.yaml]: Python dependency caveat added

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

I think I'll have to manually merge due to the broken github CI workflow.

**Pull request merged**